### PR TITLE
Make operator filter attribute not mandatory

### PR DESCRIPTION
**Pull request by juliotrigo**
- Make `op` filter attribute optional.
- It will default to `==` if not provided.

@iky here you have the improvement that you suggested. cc @mattbennett

### CODE REVIEW

File: sqlalchemy_filters/filters.py [hunk 1]
```diff
@@ -29,6 +29,9 @@ class Operator(object):
     }
 
     def __init__(self, operator):
+        if operator is None:
```
Comment (iky) [inline sqlalchemy_filters/filters.py]: also testing the `operator` simply for true value would be enough, no need for explicit `None` test .)

**Comment by juliotrigo** [comment]
@iky I've addressed your comments.
I've also updated the `CHANGELOG` so that we can include both these and the changes from @timbu from #8 in the next release .
